COM-588 Implement LTFU report

tmvumbi2 opened the pull request:
Implements LTFU report
Jira ticket: https://jembiprojects.jira.com/browse/COM-588

tmvumbi2 committed:
COM-588 Implement LTFU report
Implements LTFU report

tmvumbi2 committed:
COM-588 Refactor: re-use existing function
Renames patientWasOnARVTreatmentAtEndReportingPeriod to patientWasOnARVTreatmentByDate (more generic) and uses it in LTFU report

lauravignoli commented on reportssql/ltfu_report_util_functions.sql:
```diff
@@ -0,0 +1,128 @@
+    SELECT o.scheduled_date INTO result
+    FROM orders o
+    JOIN drug_order do ON do.order_id = o.order_id
+    JOIN concept c ON do.duration_units = c.concept_id AND c.retired = 0
```
@tmvumbi2 This join is never used? **JOIN concept c ON do.duration_units = c.concept_id AND c.retired = 0**

lauravignoli committed:
COM-588 Remove unused join

lauravignoli committed:
COM-588 Merge ltfu report files

From now on, let's have the indicator and util functions together for each report.

lauravignoli committed:
COM-588 Merge pmtct and testing report files

From now on, let's have the indicator and util functions together for each report.
